one-to-one-scheduler/one-to-one-scheduler.gs

```diff
@@ -368,6 +386,12 @@ function createMeeting(calendar, details) {
     .setGuestsCanSeeGuests(true);
 
     result.eventId = event.getId().replace("@google.com", "");
+    let eventResource = getEventResource(calendar.getId(), result.eventId);
+    if (!eventResource) {
+      result.errorMsg = "Event could not be retrieved - " + calendarId + ", " + meetId;
```

Comment:
Even if `getEventResource` fails, the event has already been created. It is just that we couldn't get its link. So, do not return error. Instead we should have the following.
```js
if (eventResource) {
    result.eventLink = eventResource.hangoutLink;
}
```

Reply:
Updated